overall-equipment-effectiveness/workflows/Paper_Machine_Data_Simulator.yaml

```diff
@@ -0,0 +1,1969 @@
+resourceType: Flow
```

Comment:
1. Let's capitalize and space the name of the node ... "Device: Get Node"
2. its, not it's
3. Capitalize "Virtual Button" in the note above this one too.

<img width="605" alt="Screen Shot 2020-09-25 at 6 33 09 PM" src="https://user-images.githubusercontent.com/881769/94321465-941a1480-ff5d-11ea-997c-8a13fddd8931.png">

Reply:
Updated.